### PR TITLE
Update links

### PR DESCRIPTION
**Pull request by richlander**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,11 +4,11 @@ The following [.NET releases](./releases.md) are currently supported:
 
 |  Version  | Release Date | Support | Latest Patch Version | End of Support |
 | :-- | :-- | :-- | :-- | :-- |
-| [.NET 8](release-notes/8.0/README.md) | [November 8, 2023](https://devblogs.microsoft.com/dotnet/announcing-dotnet-8-preview-4/) | [LTS][policies] | [8.0.0-preview.7][8.0.0-preview.7] | November 10, 2026 |
+| [.NET 8](release-notes/8.0/README.md) | [November 8, 2023](https://devblogs.microsoft.com/dotnet/announcing-dotnet-8-preview-7/) | [LTS][policies] | [8.0.0-preview.7][8.0.0-preview.7] | November 10, 2026 |
 | [.NET 7](release-notes/7.0/README.md) | [November, 2022](https://devblogs.microsoft.com/dotnet/announcing-dotnet-7/) | [STS](/release-policies.md) | [7.0.10][7.0.10] | May 14, 2024 |
```
Comment (mairaw) [inline README.md:8]:
```suggestion
| [.NET 7](release-notes/7.0/README.md) | [November, 2022](https://devblogs.microsoft.com/dotnet/announcing-dotnet-7/) | [STS](policies) | [7.0.10][7.0.10] | May 14, 2024 |
```

File: README.md [hunk 2]
```diff
@@ -4,11 +4,11 @@ The following [.NET releases](./releases.md) are currently supported:
...
 | [.NET 6](release-notes/6.0/README.md) | [November, 2021](https://devblogs.microsoft.com/dotnet/announcing-net-6/) | [LTS](/release-policies.md) | [6.0.21][6.0.21]  | November 12, 2024 |
```
Comment (mairaw) [inline README.md:9]:
```suggestion
| [.NET 6](release-notes/6.0/README.md) | [November, 2021](https://devblogs.microsoft.com/dotnet/announcing-net-6/) | [LTS](policies) | [6.0.21][6.0.21]  | November 12, 2024 |
```

File: release-notes/README.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ The following [.NET releases](../releases.md) are currently supported:
 
 |  Version  | Release Date | Support | Latest Patch Version | End of Support |
 | :-- | :-- | :-- | :-- | :-- |
-| [.NET 8](8.0/README.md) | [November 8, 2023](https://devblogs.microsoft.com/dotnet/announcing-dotnet-8-preview-4/) | [LTS][policies] | [8.0.0-preview.7][8.0.0-preview.7] | November 10, 2026 |
+| [.NET 8](8.0/README.md) | [November 8, 2023](https://devblogs.microsoft.com/dotnet/announcing-dotnet-8-preview-7/) | [LTS][policies] | [8.0.0-preview.7][8.0.0-preview.7] | November 10, 2026 |
 | [.NET 7](7.0/README.md) | [November, 2022](https://devblogs.microsoft.com/dotnet/announcing-dotnet-7/) | [STS](/release-policies.md) | [7.0.10][7.0.10] | May 14, 2024 |
```
Comment (mairaw) [inline release-notes/README.md:8]:
```suggestion
| [.NET 7](7.0/README.md) | [November, 2022](https://devblogs.microsoft.com/dotnet/announcing-dotnet-7/) | [STS][policies] | [7.0.10][7.0.10] | May 14, 2024 |
```

File: release-notes/README.md [hunk 4]
```diff
@@ -4,7 +4,7 @@ The following [.NET releases](../releases.md) are currently supported:
...
 | [.NET 6](6.0/README.md) | [November, 2021](https://devblogs.microsoft.com/dotnet/announcing-net-6/) | [LTS](/release-policies.md) | [6.0.21][6.0.21]  | November 12, 2024 |
```
Comment (mairaw) [inline release-notes/README.md:9]:
```suggestion
| [.NET 6](6.0/README.md) | [November, 2021](https://devblogs.microsoft.com/dotnet/announcing-net-6/) | [LTS][policies] | [6.0.21][6.0.21]  | November 12, 2024 |
```

File: README.md [hunk 5]
```diff
@@ -4,11 +4,11 @@ The following [.NET releases](./releases.md) are currently supported:
 
 |  Version  | Release Date | Support | Latest Patch Version | End of Support |
 | :-- | :-- | :-- | :-- | :-- |
-| [.NET 8](release-notes/8.0/README.md) | [November 8, 2023](https://devblogs.microsoft.com/dotnet/announcing-dotnet-8-preview-4/) | [LTS][policies] | [8.0.0-preview.7][8.0.0-preview.7] | November 10, 2026 |
+| [.NET 8](release-notes/8.0/README.md) | [November 8, 2023](https://devblogs.microsoft.com/dotnet/announcing-dotnet-8-preview-7/) | [LTS][policies] | [8.0.0-preview.7][8.0.0-preview.7] | November 10, 2026 |
```
Comment (mairaw) [inline README.md:7]:
```suggestion
| [.NET 8](release-notes/8.0/README.md) | [November 2023](https://devblogs.microsoft.com/dotnet/announcing-dotnet-8-preview-7/) | [LTS][policies] | [8.0.0-preview.7][8.0.0-preview.7] | November 10, 2026 |
```

Comment (richlander) [reply README.md:7]: I think we can reasonably write 11/14 per https://www.dotnetconf.net/

Comment (mairaw) [reply README.md:7]: I wasn't sure about that. Let me update it.

**Review by mairaw — APPROVED** [review]

**Pull request merged**